### ISSUE TITLE
ranger can't preview images with ueberzug

### ISSUE THREAD

**Issue by y2878463032** [ranger/ranger#2875]
#### Runtime Environment

- Operating system and version: arch 6.4.1
- Terminal emulator and version: alacritty 0.12.2
- Python version: 3.11.3
- Ranger version/commit: 1.9.3
- Locale: zh_cn

#### Question
My ranger can't use ueberzug to preview images.

There is also no corresponding preview image generated under .cache/ranger when trying to preview an image.

I have tried reinstalling ranger and ueberzug using pacman to no avail.

What should I do?

my rc.conf：
```
set preview_images true
set preview_images_method ueberzug
set w3m_delay 0.10
set w3m_offset 0
set preview_max_size 10484760
```
and image is
![image](https://github.com/y2878463032/drew-bed/blob/main/2023-07-15-21-16-13.jpg?raw=true)

**Comment by vejkse** [comment]
This was already signaled here: https://github.com/ranger/ranger/issues/2871.  In the meantime, this was fixed on ueberzug’s side in version 18.2.2, which is on ArchLinux since yesterday.

**Comment by y2878463032** [comment]
Thanks. Problem solved.
